px-vis-event.html

```diff
@@ -357,6 +361,26 @@
         .on('mouseenter', this._requestTooltip.bind(this,d,i,iconSvg.node()))
         .on('mouseleave', this._clearTooltipRequest.bind(this,d,i));
     },
+  
+    /**
+    * checks if the text property exists and adds 
+    * text to the element with the class event-text
+    *
+    * @method _enterText
+    */
+    _enterText: function(d, i, elem) {
+      const text = this._getConfigValue('text', d.label, false);
+      let iconSvg;
+
+      if(text) {
+        iconSvg = Px.d3.select(elem).append("text")
+        .text(text)
+        .attr('class', 'event-text')
+        .attr('font-size', '12px')
+        .style('font-family',this._checkThemeVariable("--px-vis-font-family", 'Comic Sans MS'))
```

Comment:
Comic Sans? 😱 

```suggestion
        .style('font-family', this._checkThemeVariable('--px-vis-font-family', 'GE Inspira Sans'))
```

Reply:
@evanjd  as far as I see the comic Sans is the default value in case --px-vis-font-family has nothing or has not loaded yet